analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -420,29 +414,28 @@ def version_compatible(cls, configured_binary, environ):
         return True
 
     def construct_result_handler(self, buildaction, report_output,
-                                 checker_labels, skiplist_handler):
+                                 skiplist_handler):
         """
         See base class for docs.
         """
         res_handler = ClangSAResultHandler(buildaction, report_output,
                                            self.config_handler.report_hash)
 
-        res_handler.checker_labels = checker_labels
         res_handler.skiplist_handler = skiplist_handler
 
         return res_handler
 
     @classmethod
-    def construct_config_handler(cls, args, context):
+    def construct_config_handler(cls, args):
 
-        environ = env.extend(context.path_env_extra,
-                             context.ld_lib_path_extra)
+        context = analyzer_context.get_context()
+        environ = context.analyzer_env
 
         handler = config_handler.ClangSAConfigHandler(environ)
```

Comment:
Just a question. Could the env parameter factored out from the Config Handler the same way?